apps/state/lib/state/service.ex

```diff
@@ -54,10 +59,15 @@ defmodule State.Service do
   end
 
   @impl State.Server
-  def handle_new_state({calendar, calendar_dates}) do
+  def handle_new_state({
+        calendar,
+        calendar_attributes,
+        calendar_dates
+      }) do
     service_ids =
       [
         calendar |> Enum.map(&Map.get(&1, :service_id)),
+        calendar_attributes |> Enum.map(&Map.get(&1, :service_id)),
         calendar_dates |> Enum.map(&Map.get(&1, :service_id))
```

Comment:
I realize you're matching the existing style here, but these shouldn't be using pipes as there's only a single one. `Enum.map(calendar_attributes, & &1.service_id)`